docs/experiments-msmarco-doc.md

```diff
@@ -212,3 +212,4 @@ That's it!
 + Results replicated by [@KaiSun314](https://github.com/KaiSun314) on 2021-01-08 (commit [`113f1c7`](https://github.com/castorini/anserini/commit/113f1c78c3ffc8681a06c571901cf9ad8f5ee633))
 + Results replicated by [@yemiliey](https://github.com/yemiliey) on 2021-01-18 (commit [`179c242`](https://github.com/castorini/anserini/commit/179c242562bbb990e421f315370f34d4d19bbb9f))
 + Results replicated by [@larryli1999](https://github.com/larryli1999) on 2021-01-22 (commit [`3f9af5`](https://github.com/castorini/anserini/commit/3f9af54d6215eacbade7fc99ff8890920fdddee0))
++ Results replicated by [@ArthurChen189](https://github.com/ArthurChen189) on 2021-04-08 (commit [`d43b5c6`](https://github.com/castorini/anserini/commit/b0e0092fb4c4f2bf053c9e5d35b63aca6fd30e47))
```

Comment:
This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.